bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+
+After reading the information in the above sections to setup HLS, you can ask Google to 'Show the Front Door Camera' with the recently added metadata for the Openhab Cloud Connector.
+ 
+Don't forget to ask google to 'sync my devices' after adding the metadata shown below. 
+You can also ask for any of the synonyms that you tag to ensure Google understands multiple names that the camera may be called by different people in your family.
+
+Example of how this is done in your items file.
+
+```
+
+String FrontDoorCamHlsUrl "Front Door" { channel="ipcamera:ONVIF:FrontDoor:hlsUrl", synonyms="door bell, front camera", ga="Camera" [ protocols="hls" ] }
+
+```
+
+How to cast the camera from inside a rule using the Chromecast binding:
+
+items
+
+```
+String KitchenHomeHubPlayURI { channel="chromecast:chromecast:KitchenHomeHub:playuri" }
+
+```
+
+
+In a rule...
+
+```
+
+KitchenHomeHubPlayURI.sendCommand("http://192.168.1.2:54321/ipcamera.m3u8")
+
+```
+
+
+
+## HABpanel
+
+This section is about how to get things working in Habpanel.
+
+I highly recommend you check out the easy to use WIDGETS of which there are now 3 ready made ones that are discussed on the forum here.
+<https://community.openhab.org/t/custom-widget-camera-clickable-thumbnails-that-open-a-stream/101275>
+
+The widgets in the link above are the easiest way to get an advanced stream working in Openhab and you are welcome to open them up, look at how they work and change them to something even better that suits your needs.
+If you don't like doing things the easy way with a ready made widget, below are how it can be done without a widget.
+
+**How to manually display Mjpeg based streams without using the above WIDGETS:**
+
++ Add a FRAME widget
++ Set URL Source to be 'openHAB String Item'
++ Select the item that is bound to the ``streamUrl`` channel of your camera that is setup with this binding and ONLINE.
++ Alternatively you can set a static url like ``http://192.168.1.2:50001/snapshots.mjpeg`` to have the binding create a high resolution mjpeg stream out of your snapshots or the cameras url to fetch the stream directly.
+
+
+**How to manually display HLS without using the above WIDGETS:**
+
++ Add a template widget with the following code using an Openhab item to link to your cameras ``hlsUrl`` channel.
+
+
+```
+
+<video width="100%" height="100%" autoplay src="{{itemValue('Camera_hlsUrl')}}"</video>
+
+```
+
+
+
+## Animated GIF
+
+This binding has a channel called `updateGif` and when this switch is turned 'ON' (either by a rule or manually) the binding will create an animated GIF called ipcamera.gif in the ffmpeg output folder.
+You can change the filename using the channel that is called `gifFilename` and an example of how to use this in a rule can be seen under the MP4 recording section. Both features are very similar.
```

Comment:
You could combine these chapters to shorten this extensive document a bit.

Reply:
Shortened them both and made them follow so if someone reads in order the info translates over.